scripts/generate_reviews_files.py

```diff
@@ -84,6 +86,19 @@ def make_column_unique(df, column_name):
     df[column_name] = new_values
     return df
 
+# Function that fills the "Variant Called by CLE Pipeline" column based on the matching variant values
+# for "VALIDATED" in the variants.final.annotated.tsv file
+def fill_variant_called_column(df, variants):
```

Comment:
This will work For single nucleotide variants, but I think will fail to match both insertions and deletions...

Reply:
Here is an example of the two representations for a deletion:
chr18	24171433	ACTTTGTGTAATTC	A
chr18-24171433-24171446-ACTTTGTGTAATTC-A

Here is an example of the two representation for an insertion.
chr6	115943027	A	AGT
chr6-115943027-115943027-A-AGT